Makefile

```diff
@@ -437,7 +437,7 @@ test: clean-launch-dev launch-dev test-misc test-pps
 enterprise-code-checkin-test:
 	# Check if our test activation code is anywhere in the repo
 	@echo "Files containing test Pachyderm Enterprise activation token:"; \
-	if grep --files-with-matches --exclude=Makefile --exclude-from=.gitignore -r -e 'RM2o1Qit6YlZhS1RGdXVac' . ; \
+	if grep --files-with-matches --exclude=Makefile --exclude-dir=.git -r -e 'RM2o1Qit6YlZhS1RGdXVac' . ; \
```

Comment:
It seems like semantics here are slightly different, does osx grep not have `--exclude-from`?

Reply:
It does not

Reply:
One alternative I could pursue is to just hardcode in all of the excludes specified on `.gitignore`. I didn't go with this originally because:

* It seems a bit brittle - subsequent changes to `.gitignore` would also have to be reflected here
* In practice, this `grep` call runs pretty fast, especially once `.git` is ignored

Reply:
I see, @msteffen added this originally I think. My sense is that there's a lot of things in our `.gitignore` that often contain the thing we're worried about. So it'll trigger a lot during development.